src/Cmfd.cpp

```diff
@@ -139,7 +107,7 @@ void Cmfd::setNumY(int num_y) {
 
 /**
  * @brief Get the number of Mesh cells in a row.
- * @return number of Mesh cells in a row
+ * @return The number of Mesh cells in a row
```

Comment:
Two questions - first, are a lot of these changes to the Doxygen comments to address #117 or just general cleanup? Second, it looks like you capitalize the first word in the parameter comments in many but not all of the Doxygen comment blocks. At this point, I tend to think that capitalization is best for the documentation, though this is not consistent throughout the code. I don't see this as an issue at all for this PR, but it might be something that we should come to an agreement upon and add to the Style Guide to be consistent in the future.